lib/matplotlib/rcsetup.py

```diff
@@ -23,6 +23,8 @@
 from matplotlib.fontconfig_pattern import parse_fontconfig_pattern
 from matplotlib.colors import is_color_like
 
+from cycler import cycler, Cycler
```

Comment:
The `Cycler` class and the (recommended to use) `cycler()` function are available directly from the top level module.